google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/SelectConfigDestinationFolderDialog.java

```diff
@@ -60,4 +63,15 @@ protected JComponent createCenterPanel() {
   public File getDestinationFolder() {
     return new File(destinationFolderChooser.getText());
   }
+
+  @NotNull
+  protected Action[] createActions() {
+    getOKAction().putValue(Action.NAME, "Generate");
+
+    if (SystemInfo.isMac) {
```

Comment:
On second thought, I see that the superclass does this. Should there be an @Override annotation here?